frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/HiddenExpressionOnLayout.test.tsx

```diff
@@ -22,7 +22,7 @@ const defaultLayouts: IFormLayouts = {
   [layout1NameMock]: layoutMock,
 };
 
-jest.mock('app-shared/hooks/useDebounce', () => ({
+jest.mock('../../../../../../libs/studio-components/src/hooks/useDebounce.ts', () => ({
```

Comment:
I believe that you used the relative path to avoid mocking/conflicting with other modules from studio-components that are used within this component. What about using an alias import anyway? I think this should work, but not sure! 👼 

```suggestion
jest.mock('@studio/components/src/hooks/useDebounce.ts', () => ({
```

Reply:
I noticed jest.config was missing moduleMapping for this type of import, so i have added regex to match the path.